cmd/nginx/flags.go

```diff
@@ -156,6 +156,9 @@ Feature backed by OpenResty Lua libraries. Requires that OCSP stapling is not en
 		sslProxyPort  = flags.Int("ssl-passthrough-proxy-port", 442, `Port to use internally for SSL Passthrough.`)
 		defServerPort = flags.Int("default-server-port", 8181, `Port to use for exposing the default server (catch-all).`)
 		healthzPort   = flags.Int("healthz-port", 10254, "Port to use for the healthz endpoint.")
+
+		disableCatchAll = flags.Bool("disable-catch-all", true,
```

Comment:
I'm not sure if we should be disabling this by default. I think it's better to keep the current behaviour and support catch all ingresses by default.

Reply:
Sorry, this will be `false` by default. This was for my testing